docs/docs/integrations/text_embedding/solar.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "0f1199c1-f885-4290-b5e7-d1defd49abe1",
    "metadata": {},
    "source": [
-    "# Soalr\n",
+    "# Solar\n",
```

Comment:
I know, you didn't change this part but it would be easy to you fix this.
Here is the printing of the whole vectors (in two places) which is not necessary but makes the notebook unreadable. Instead, print just `len(query_result)` and `len(document_result)` 
It would be highly appreciated :) Thanks!